x-pack/test/api_integration/apis/ml/data_frame_analytics/post.ts

```diff
@@ -0,0 +1,158 @@
+      await createJobs(testJobConfigs);
+    });
+
+    after(async () => {
+      await ml.api.cleanMlIndices();
+    });
+
+    describe('StartDataFrameAnalyticsJob', () => {
+      it('should start analytics job for specified id if job exists', async () => {
+        const analyticsId = `${jobId}_0`;
+
+        const { body } = await supertest
+          .post(`/api/ml/data_frame/analytics/${analyticsId}/_start`)
+          .auth(USER.ML_POWERUSER, ml.securityCommon.getPasswordForUser(USER.ML_POWERUSER))
+          .set(COMMON_REQUEST_HEADERS)
+          .expect(200);
+
+        expect(body).not.to.be(undefined);
+        expect(body.acknowledged).to.be(true);
+        expect(body.node).not.to.be('');
+      });
+
+      it('should show 404 error if job does not exist', async () => {
+        const id = `${jobId}_invalid`;
+        const message = `No known job with id '${id}'`;
+
+        const { body } = await supertest
+          .post(`/api/ml/data_frame/analytics/${id}/_start`)
+          .auth(USER.ML_POWERUSER, ml.securityCommon.getPasswordForUser(USER.ML_POWERUSER))
+          .set(COMMON_REQUEST_HEADERS)
+          .expect(404);
+
+        expect(body.error).to.eql('Not Found');
+        expect(body.message).to.eql(message);
+      });
+
+      it('should not allow to start analytics job for unauthorized user', async () => {
+        const analyticsId = `${jobId}_0`;
+
+        const { body } = await supertest
+          .post(`/api/ml/data_frame/analytics/${analyticsId}/_start`)
+          .auth(USER.ML_UNAUTHORIZED, ml.securityCommon.getPasswordForUser(USER.ML_UNAUTHORIZED))
+          .set(COMMON_REQUEST_HEADERS)
+          .expect(403);
+
+        expect(body.error).to.eql('Forbidden');
+        expect(body.message).to.eql('Forbidden');
+      });
+    });
+
+    describe('StopsDataFrameAnalyticsJob', () => {
+      it('should stop analytics job for specified id when job exists', async () => {
+        const analyticsId = `${jobId}_0`;
+
+        const { body } = await supertest
+          .post(`/api/ml/data_frame/analytics/${analyticsId}/_stop`)
+          .auth(USER.ML_POWERUSER, ml.securityCommon.getPasswordForUser(USER.ML_POWERUSER))
+          .set(COMMON_REQUEST_HEADERS)
+          .expect(200);
+
+        expect(body).not.to.be(undefined);
+        expect(body.stopped).to.be(true);
+      });
+
+      it('should show 404 error if job does not exist', async () => {
+        const id = `${jobId}_invalid`;
+        const message = `No known job with id '${id}'`;
+
+        const { body } = await supertest
+          .post(`/api/ml/data_frame/analytics/${id}/_stop`)
+          .auth(USER.ML_POWERUSER, ml.securityCommon.getPasswordForUser(USER.ML_POWERUSER))
+          .set(COMMON_REQUEST_HEADERS)
+          .expect(404);
+
+        expect(body.error).to.eql('Not Found');
+        expect(body.message).to.eql(message);
+      });
+
+      it('should not allow to stop analytics job for unauthorized user', async () => {
```

Comment:
Would be good to have another test that asserts the same for a user with viewer permissions (ML_VIEWER).